config/packages/doctrine_migrations.yaml

```diff
@@ -1,5 +1,3 @@
 doctrine_migrations:
-    dir_name: '%kernel.project_dir%/src/Migrations'
-    # namespace is arbitrary but should be different from App\Migrations
-    # as migrations classes should NOT be autoloaded
-    namespace: DoctrineMigrations
+    migrations_paths:
+        DoctrineMigrations: '%kernel.project_dir%/src/Migrations'
```

Comment:
It is advised to put migrations outside the src directory

Reply:
Good point. Added a chore ticket to pivotal

https://www.pivotaltracker.com/story/show/187430536